### DISCUSSION TITLE
Live Stream intermittently, GP 9?

### DISCUSSION THREAD

**Discussion by piersg** [gopro/labs#207] [Q&A] [answered]
Hi There,

I'm trying to run some tests with the ultimate goal being a GP that live streams only during daylight hours (it will be looking out a window at a scenic beach)

So I could use conditional time triggers, or 'r' and 's', but to test the general functionality, I'm trying to get it to stream for a minute, not stream for a minute, then repeat. And it's not quite working.

The repeating and the waiting are fine, but no matter how long I put before the 'E' command, it only streams for about 9 seconds.

The command I'm using is:

!W!GM!300E!60R

(so in that instance, I'm trying to wait 300s before I stop streaming)

Hopefully I am just missing something simple ...

**Comment by piersg** [comment, marked as answer]
If I put a "nop" in, i.e. instead of 

`!300E`

I do 

`!300N!E`

It seems to work with a big "busy" across the screen, which I can live with if it's the only way

![image](https://user-images.githubusercontent.com/505856/213976011-15d737a8-ea3c-4c7a-8117-fcd6a6c60feb.png)

**Reply by dnewman-gpsw** [reply]
The end command !E  is the counter to a !S start video, but live streaming doesn't work that way.  Seems you have found an okay work around.
